auth-service/src/test/kotlin/uk/gov/communities/delta/controllers/EditUserAccessGroupsControllerTest.kt

```diff
@@ -235,13 +237,122 @@ class EditUserAccessGroupsControllerTest {
             }
         }.apply {
             assertEquals("nonexistent_group", errorCode)
-            assertEquals(HttpStatusCode.Forbidden, statusCode)
+            assertEquals(HttpStatusCode.BadRequest, statusCode)
             coVerify(exactly = 0) { groupService.addUserToGroup(any(), any(), any(), any(), any()) }
             coVerify(exactly = 0) { groupService.removeUserFromGroup(any(), any(), any(), any(), any()) }
             confirmVerified(groupService)
         }
     }
 
+    @Test
+    fun userCanAddOtherToAccessGroup() = testSuspend {
+        testClient.post("/access-groups/add") {
+            headers {
+                append("Authorization", "Bearer ${internalUserSession.authToken}")
+                append("Delta-Client", "${client.clientId}:${client.clientSecret}")
+            }
+            contentType(ContentType.Application.Json)
+            setBody(
+                "{" +
+                    "\"userToEditCn\": \"${externalUser.cn}\"" +
+                    ", \"accessGroupName\": \"access-group-3\"" +
+                    "}"
+            )
+        }.apply {
+            assertEquals(HttpStatusCode.OK, status)
+            coVerify(exactly = 1) {
+                groupService.addUserToGroup(
+                    externalUser.cn,
+                    externalUser.dn,
+                    "datamart-delta-access-group-3",
+                    any(),
+                    internalUserSession
+                )
+            }
+            coVerify(exactly = 0) {
+                groupService.removeUserFromGroup(
+                    any(),
+                    any(),
+                    any(),
+                    any(),
+                    any()
+                )
+            }
```

Comment:
Fine to keep it for consistency with other tests if you like, but the `confirmVerified` makes it unnecessary